proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,496 @@
+<!-- [TBD] open pull requests are directly related to an existing feature request. -->
+
+We propose to manage existing pull requests in two phases. First, we will
+determine if there is a corresponding tracking issue for each pull request and
+[link them
+together](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
+If no tracking issue exists, we will create one and link it to the pull request.
+Then, we will apply the new specification to all open issues identify the
+current stage. We will prioritize the review of issues with a linked pull
+request. After review, all pull requests will follow the process and timelines
+defined in the specification.
+
+## Rationale/FAQ
+
+### Q. Why do we limit the number of issues in the review stage?
+
+We do not want to discourage users from contributing or proposing changes, but
+we need to acknowledge that the maintainers have limited bandwidth to give
+feedback on implementation or design and review pull requests. The maintainers
+have work that comes from outside of the GitHub repository that needs to be
+prioritized along with user contributions. Controlling the volume of pull
+request reviews and prioritizing completing the in flight reviews over new
+reviews will result in a shorter time to merge and release a contributor's
+change for a feature requested by the community. It will also reduce the amount
+of context switching for the maintainers, allowing them to focus on giving high
+quality feedback to users. As we use this framework to process existing feature
+requests, we can revisit what the in flight limit should be.
+
+<a name="rationale-require-issues"></a>
+### Q. Why do we require issues to be opened instead of just pull requests?
+
+Having a defined process for intake removes any ambiguity on how to initiate a
+contribution by ensuring that all potential contributions start out as a GitHub
+issue. We feel that having a more general discussion about the problem and
+solutions will help reduce the overall effort into code review and provide
+contributors with enough detail to make a sucessful contribution.
+
+This requirement also solidifies the position that all pull requests that are
+currently open are something we'd like to pull in. It removes any ambiguity for
+a contributor about the status of the request.
+
+Pull requests are often made for problems that only affect the contributor or a
+small portion of the user base. An issue provides a mechanism to gather
+quantitative feedback in the form of "upvotes" through GitHub reactions to
+estimate the impact of the issue on the community.
+ 
+We intend for issues to be a way to improve contributor confidence in both their
+contributions and the overall process. If a user does open a pull request
+without an issue, we will use the opportunity to educate them about our process,
+and when necessary assist them in opening an issue.
+
```

Comment:
I do think we should make a note somewhere that it is totally fine to write up a branch/draft PR to illustrate the request and this branch/draft PR can be converted to an actual PR.